bin/git-standup

```diff
@@ -155,7 +160,7 @@ fi
 AUTHOR=${AUTHOR:="$(git config user.name)"}
 FETCH_LAST_COMMIT=${FETCH_LAST_COMMIT:=false}
 MAXDEPTH=${MAXDEPTH:=2}
-GIT_PRETTY_FORMAT="%Cred%h%Creset - %s %Cgreen(%cd) %C(bold blue)<%an>%Creset $GIT_PRETTY_FORMAT"
+GIT_PRETTY_FORMAT="%Cred%h%Creset - %s %Cgreen(${COMMIT_DATE_FORMAT}) %C(bold blue)<%an>%Creset $GIT_PRETTY_FORMAT"
```

Comment:
IMHO, it would be better to configure the format in one option, instead of separate options like `-g` / `-A`.

Reply:
Sorry for late reply, I missed this comment.
Should we them remove to `-g` option?